pkgs/by-name/ms/msnake/package.nix

```diff
@@ -18,6 +18,12 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-5q3yT7amPF+SSvO6/eUU7IiK0k6f3nme9YYBUobSuuo=";
   };
 
+  postPatch = ''
+    substituteInPlace CMakeLists.txt --replace-fail \
+      "cmake_minimum_required(VERSION 2.8.4)" \
+      "cmake_minimum_required(VERSION 3.5)"
```

Comment:
Can you consider bumping this to cmake 3.5 instead? This is recommended within the tracking pr